pages/confirmEmail.js

```diff
@@ -0,0 +1,38 @@
+import Head from 'next/head';
+import confirmEmailStyles from '@app/styles/confirmEmail.module.css';
```

Comment:
Nit: Gotta double-check this with Pablo but I believe the UpperCamelCase convention is a more appropriate and consistent way to name your imported styles in React

```suggestion
import ConfirmEmailStyles from '@app/styles/confirmEmail.module.css';
```

Reply:
+1